### DISCUSSION TITLE
WFO MAF endpoints returning 503 error

### DISCUSSION THREAD

**Discussion by Yeshia** [weather-gov/api#317] [Q&A]
The MAF endpoints  api.weather.gov/gridpoints/MAF/61,110/forecast and  api.weather.gov/gridpoints/MAF/105,127/forecast/hourly
both are returning 503 "The resource you requested is currently unavailable.  Please try again later"

These are the urls provided by the api.weather.gov/points/latlong  response.

**Comment by KTibow** [comment]
I'm not offical here, but I have notifications turned on, and I think that you should probably check the other issues first, including the closed ones. Check #76 and https://weather-gov.github.io/api/reporting-issues
